docs/home/contribute/localization.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+
+### Basis for localizations
+
+Localizations must be performed on the most recent major release (for example, 1.8). The English source for major releases can be found in [TODO].
```

Comment:
@zacharysarah For now, let's say that the English source should come from the branch of the last major release. You can use the page variable `{{page.version}}` to display that version number. They should then go to the relevant branch `release-1.x` and use the English docs files in the `/docs/` directory. Eventually, I would like to move this into `/i18n/en/docs/...`, but for now, we should stick w/ our current structure.

Reply:
@chenopis Just found this in a stray window and realized I never followed up on it. 😳 Making updates now.